charts/cloudserver-front/values.yaml

```diff
@@ -18,7 +18,7 @@ mongodb:
 replicaCount: 1
 image:
   repository: zenko/cloudserver
-  tag: 0.1.4
+  tag: 0.1.5
```

Comment:
Over time, changes like these will also require to bump the version in `Chart.yaml`, but I guess we're good for now.

Reply:
yes you're right, I just tested it but I will bump the charts number as well.

Reply:
Note bumping the number will not be sufficient. From memory:

- Bump the number in cloudserver-front's `Chart.yaml`
- Bump the requirement version in zenko's `requirements.yaml`
- Rebuild zenko's `requirement.lock` through `helm dep update`